src/utils/summarizeTests.ts

```diff
@@ -2,12 +2,12 @@ import { ComparisonResult, StoryTestFieldsFragment, TestResult, TestStatus } fro
 import { aggregateResult } from "./aggregateResult";
 
 function pickStatus(statusCounts: { [K in TestStatus]?: number }) {
+  if (statusCounts[TestStatus.InProgress] > 0) return TestStatus.InProgress;
   if (statusCounts[TestStatus.Failed] > 0) return TestStatus.Failed;
```

Comment:
Maybe Failed should be before InProgress at least? That's the way it works on the Chromatic side: https://github.com/chromaui/chromatic/blob/6da7d96b3c7eaa8383616bee2665cd54a118d6fc/services/index/model/Test.ts#L425-L433